torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4293,22 +4293,53 @@ def shape(size, rank, with_batch_channel=True):
             return tuple([N, C] + ([size] * rank))
         return tuple([size] * rank)
 
-    make_arg = partial(make_tensor, device=device, dtype=dtype,
-                       requires_grad=requires_grad, low=-1, high=1)
+    if mode in ('bilinear', 'bicubic'):
+        make_arg = partial(
+            make_tensor,
+            device=device,
+            dtype=dtype,
+            requires_grad=requires_grad,
+            # we pick more realistic upper bound 256 instead of default 10 for uint8 dtype
+            high=256 if dtype == torch.uint8 else None,
+        )
+        # provide a single sample for more typical image processing usage
+        rank = 2
+        yield SampleInput(
+            make_arg(shape(400, rank)),
+            shape(270, rank, False),
```

Comment:
Increasing the first dim from 5 to 400 probably causing the memory increase. We might be able to reland with this as a reference input instead of a sample.